package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "wmt-worker",
+  "version": "0.1.0",
+  "description": "WMT Async worker application",
+  "main": "start.js",
+  "repository": "https://github.com/ministryofjustice/wmt-worker.git",
+  "author": "Tom Swann",
+  "license": "MIT",
+  "scripts": {
+    "start": "node start.js",
```

Comment:
Worker doesn't really have a need for it yet I think - unlike the web app which has assets to build and similar.